tests/src/Functional/ContentPersonRenderTest.php

```diff
@@ -347,8 +347,12 @@ public function testPersonRendering(): void {
     $this->assertCount(2, $transparency_links_items);
     $this->assertEquals('http://example.com/link_1', $transparency_links_items[0]->getAttribute('href'));
     $this->assertEquals('Person link 1', $transparency_links_items[0]->getText());
+    $first_link_icon = $transparency_links_items[0]->find('css', 'a.ecl-link svg.ecl-icon.ecl-icon--s.ecl-icon--primary.ecl-link__icon');
+    $this->assertEquals('<use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="/build/themes/custom/oe_theme/dist/ec/images/icons/sprites/icons.svg#external"></use>', $first_link_icon->getHtml());
     $this->assertEquals('http://example.com/link_2', $transparency_links_items[1]->getAttribute('href'));
     $this->assertEquals('http://example.com/link_2', $transparency_links_items[1]->getText());
+    $second_link_icon = $transparency_links_items[0]->find('css', 'a.ecl-link svg.ecl-icon.ecl-icon--s.ecl-icon--primary.ecl-link__icon');
```

Comment:
Are we checking again same link?
```suggestion
    $second_link_icon = $transparency_links_items[1]->find('css', 'a.ecl-link svg.ecl-icon.ecl-icon--s.ecl-icon--primary.ecl-link__icon');
```